### ISSUE TITLE
Missing CSS for Post Info with "Improved CSS Loading" enabled

### ISSUE THREAD

**Issue by ShadowXVII** [elementor/elementor#15658]
Items in a flattened Post Info box show fine during editor
![image](https://user-images.githubusercontent.com/446329/125740264-733b1c23-6598-4bc3-a44b-1781bbcaef79.png)

```
.elementor-widget .elementor-icon-list-items {
    list-style-type: none;
}
```

However, when published and Improved CSS loading is on, it doesn't hide the list style.

![image](https://user-images.githubusercontent.com/446329/125740519-fa0444c7-88ea-44d9-9ba8-32a9f1152374.png)

Elementor - 3.3.0
Elementor Pro - 3.3.2

**Comment by DorShahar** [comment]
Thanks for reporting.
We are aware of this use case - I'm merging the issue with #15632, which is on the same subject so we could deal with it more efficiently. 

Please feel free to continue the conversation there.
Thanks for your feedback!

**Comment by ulrichson** [comment]
@DorShahar This issue is still present in Elementor 3.4.2, also after re-generating CSS files. Could this issue be re-opened (the merged issue #15632 is already closed)?

**Comment by ShadowXVII** [comment]
@DorShahar Yep, this hasn't been fixed -- in-fact I've more issues now with headings. I'll create a new issue later if I don't hear anything.
